lib/send-metrics-data.js

```diff
@@ -1,7 +1,14 @@
-/* global Services */
+const Metrics = require('testpilot-metrics');
+const store = require('sdk/simple-storage').storage;
+const manifest = require('../package.json');
 
-const {Cu} = require('chrome');
-Cu.import('resource://gre/modules/Services.jsm');
+const { sendEvent } = new Metrics({
+  id: manifest.name,
```

Comment:
I was wrong - turns out you do need the '@' symbol. I think this is the nicest way to get it: `require('sdk/self').id`.